packages/fx-core/resource/package.nls.json

```diff
@@ -661,6 +661,8 @@
   "error.arm.ConvertArmOutputError": "Unable to convert ARM deployment result to action output, there is a duplicated key '%s' in ARM deployment result.",
   "error.deploy.DeployEmptyFolderError": "Unable to locate any files in the distribution folder: '%s'. Please ensure that the folder is not empty and that all necessary files have been included.",
   "error.deploy.CheckDeploymentStatusTimeoutError": "Unable to check deployment status because the process timed out. Check your internet connection and try again. If the issue persists, please review the deployment logs (Deployment -> Deployment center -> Logs) in Azure portal to identify any issues that may have occurred.",
+  "error.deploy.ZipFileError": "Unable to zip the distribution folder. Please ensure that the folder is not larger then 2GB.",
```

Comment:
typo: "than". Also curious about the meaning of "distribution folder".

Reply:
nice catch, changed to artifact folder to fit with the YAML file.